spec/index.bs

```diff
@@ -1830,7 +1830,21 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
     </div>
 </div>
 
-## GPUBindGroup ## {#gpu-bindgroup}
+### Compatibility ### {#bind-group-compatibility}
+
+<div algorithm>
+Two {{GPUBindGroupLayout}} objects |a| and |b| are considered <dfn dfn>equivalent</dfn> if all of the following conditions are true:
+
+  1. |a|.{{GPUBindGroupLayout/[[entries]]}}.length == |b|.{{GPUBindGroupLayout/[[entries]]}}
+  2. For each entry in |a|.{{GPUBindGroupLayout/[[entries]]}} there is an equivalent entry in |b|.{{GPUBindGroupLayout/[[entries]]}}
```

Comment:
u-unit: What happens if there are two entries foo and bar, a is [foo, foo, bar] and b is [foo, bar, bar]. This can't happen by construction, but it would be nice to call it out to explain why this algorithm works.